sormas-ui/src/main/java/de/symeda/sormas/ui/email/ExternalBulkEmailOptionsForm.java

```diff
@@ -108,6 +119,37 @@ protected void addFields() {
 		if (documentRelatedEntityType != null) {
 			attachedDocumentsField = addField(ExternalEmailOptionsWithAttachmentsDto.ATTACHED_DOCUMENTS, MultiSelectFiles.class);
 		}
+
+		DocumentWorkflow templatesWorkflow = getDocumentTemplateWorkflow();
+		if (templatesWorkflow != null) {
+			attachDocTemplateLayout = new QuarantineOrderLayout(templatesWorkflow);
+			getContent().addComponent(attachDocTemplateLayout, CUSTOM_EMAIL_ATTACHMENT_DOCUMENT);
+		}
+	}
+
+	@Nullable
+	private DocumentWorkflow getDocumentTemplateWorkflow() {
+		DocumentWorkflow templatesWorkflow;
+		switch (documentWorkflow) {
```

Comment:
this switch can be avoided if you require the generated document workflow in the constructor of this class.

Reply:
removed the switch, added the templateWorkflow as parameter in constructor